### ISSUE TITLE
Task from example project SQUAD2.0 is not built as expected (validation error in preview).

### ISSUE THREAD

**Issue by noath** [Toloka/toloka-kit#15]
The task from example project for SQUAD2.0 [notebook](https://github.com/Toloka/toloka-kit/blob/main/examples/SQUAD2.0/SQUAD2.0_processing.ipynb) is not built as expected. Performers will not be able to mark "No" option because it would cause validation error: 
![image](https://user-images.githubusercontent.com/29887838/116717595-b9826b80-a9e1-11eb-971a-5598f315b1cd.png)

Moreover it works fine in the same project with preview of template builder in web-version:
![image](https://user-images.githubusercontent.com/29887838/116717182-4a0c7c00-a9e1-11eb-9682-310a4eee1304.png)

I guess it is enough to add `required=False` for `'answer'` in output specs for project, because the check that the answer is not empty is built-in by default, but does the template builder work as intended in this case?

**Comment by varfolomeii** [comment]
Hi, thank you for reporting. Yes, the `answer` field should not be required. We've fixed it and the changes will be merged to main very soon.

**Comment by noath** [comment]
@varfolomeii and what about template builder behaviour? 
> ...does the template builder work as intended in this case?
I mean is it fine that template builder does not recognize empty required field in the preview mode?

**Comment by ortemij** [comment]
@noath Could you please share more details about the wrong behavior of TB? Steps to reproduce the issue.

**Comment by noath** [comment]
@ortemij Sure. I created the project from the [example notebook about SQUAD 2.0](https://github.com/Toloka/toloka-kit/blob/main/examples/SQUAD2.0/SQUAD2.0_processing.ipynb), then i opened web-version of Toloka and went to freshly baked project. Then i clicked on the button `Project actions` -> `Edit` and scrolled to Task Interface section [(here)](https://user-images.githubusercontent.com/29887838/116717182-4a0c7c00-a9e1-11eb-9682-310a4eee1304.png). Finally, i tried to submit task with the "no" answer in the template builder (sections Preview and Submitted data on the last screen), and it was succeeded.

But despite this success performer will face the validation error in real task (which is not remarkable from template builder editor) and in the project task preview mode (as you can see [here](https://user-images.githubusercontent.com/29887838/116717595-b9826b80-a9e1-11eb-971a-5598f315b1cd.png)).

In other words, the validation error (caused by empty required field) that occurs in preview mode and real tasks does not occur when editing the same interface using the template builder. So, is this the expected behavior?

**Comment by ortemij** [comment]
@noath It doesn't seem to be expected behavior. It would be easier for us to reproduce via the resulting template for TB, but it's manageable. We'll come back with the answer soon!

**Comment by varfolomeii** [comment]
@noath Hi, we've released a new version of the example, now everything should work correctly. Regarding the strange behaviour of the template builder interface - in short, it is expected. We will provide you more details about this a little bit later.
